stock_mts_mto_rule/model/procurement.py

```diff
@@ -19,7 +19,7 @@ class ProcurementOrder(models.Model):
     @api.multi
     def get_mto_qty_to_order(self):
         self.ensure_one()
-        stock_location = self.warehouse_id.lot_stock_id.id
+        stock_location = self.rule_id.mts_rule_id.location_src_id.id
```

Comment:
Seems coherent. Is there situation 'self.rule_id.mts_rule_id.location_src_id.id' returns False ?

Can you add a unit test for this ?

Reply:
location_src_id is mandatory in view for this type of procurement rule (action=move).
So no risk to return False. If it does, the configuration is badly wrong anyway

Reply:
Ok. Since module integrates this in v11, this can be fast one.